src/components/Overview/Overview.react.js

```diff
@@ -37,10 +51,29 @@ class Overview extends Component {
   constructor(props) {
     super(props);
     this.state = {
+      gifIndex: 0,
       isVideoModalOpen: false,
     };
+
+    this.gifArray = [
+      WebDemo,
+      LocationDemo,
+      GIFDemo,
+      JokesDemo,
+      FactsDemo,
+      MathDemo,
+    ];
+    this.buttonAttributes = [
```

Comment:
Merge this 2 arrays into one. Add gif field in the object of 2nd array. 
Declare it as a constant outside the Class declaration itself.